File: webClient/src/app/_models/order.ts
import { Customer } from './customer';
import { VacationPackage } from './vacation-package';
import { Manager } from './manager';

export class Order {
  id: number;
  price: number;
  vacationPackage: VacationPackage;
  customer: Customer;
  manager: Manager;
  creationDateTime: string;
  completedDateTime: string;
}

File: webAPI/Entities/Order.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace WebApi.Entities
{
    public class Order
    {
        public int Id { get; set; }

        [Required]
        public int Price { get; set; }

        [Required]
        public VacationPackage VacationPackage { get; set; }
        public int VacationPackageId { get; set; }

        [Required]
        public Customer Customer { get; set; }
        public int CustomerId { get; set; }

        [Required]
        public Manager Manager { get; set; }
        public int ManagerId { get; set; }

        [Required]
        public DateTime CreationDateTime { get; set; }

        [Required]
        public DateTime CompletedDateTime { get; set; }

        public bool IsDeleted { get; set; }
    }
}